app/controllers/posts_controller.rb

```diff
@@ -4,6 +4,23 @@ def index
     @posts = @user.posts
   end
 
+  def create
+    @user = User.find(params[:user_id])
+    @post = Post.new(
+      author: current_user,
+      title: params[:title],
+      text: params[:text],
+      comments_counter: 0,
+      likes_counter: 0
+    )
+
+    if @post.save
+      redirect_to user_posts_path(@user)
+    else
+      render 'new'
+    end
+  end
+
```

Comment:
- [ ] Good job creating this method in the controller, but please kindly include a `new` method that will handle requests for that route.
- [ ] [OPTIONAL] It would be better if you utilized strong parameters in defining parameters that are permitted while creating a new post. You can find the lesson on it [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)